src/foam/u2/view/UnstyledTableView.js

```diff
@@ -80,6 +80,13 @@ foam.CLASS({
         if ( ! this.of && data ) this.of = data.of;
       }
     },
+    {
+      class: 'foam.dao.DAOProperty',
+      name: 'refDAO',
+      factory: function() {
+        return this.data;
+      }
```

Comment:
How is this different than just using this.data?
Is the only real change that we are no longer doing the inX()? Could we just remove that?

Reply:
Why does it pass a NullDAO?

Reply:
Because it needs to give UnstyledTableView a NullDAO to load the headers and editColumnsView but then the pages are rendered and managed by the ScrollTableView. This issue should also be fixed with the new TableView.